### DISCUSSION TITLE
Control the order of Opportunities

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#567] [Ideas] [answered]
I would like to control the order / ranking of showing Opportunities . I think now they are always ordered alphabetically (z..a)?

**Comment by techsmyth** [comment]
Hi Gerlof, 

this is a generic item as well: what options to provide users for sorting the results shown. I have opened a new epic for this to ensure we look at it as a whole: https://github.com/alkem-io/alkemio/issues/574

For my understanding, is this something you want to be able to do as a end user, or do you also want to have some options to control sorting as an administrator of the containing Challenge?

**Reply by GerlofdeHaan** [reply]
Hi Neal, I guess both. In this specific case I want to control sorting as an administrator.

**Reply by techsmyth** [reply]
Thanks Gerlof. 

One more question: would something like being able to "pin" opportunities to the top of the list be suitable? Sorting order in absolute terms may not be intuitive. But being able to "mark" / "pin" ones that should be available first might be what you are after. In that case they would then always be at the top of the list. 

You can see this idea in action here: https://github.com/alkem-io - where I have "pinned" some repositories to the top so they always appear in that section. 

I think this may need a mockup to discuss some options. 

Fijne avond, Neil

**Reply by GerlofdeHaan** [reply]
Hi Neil,
This would be suitable for now. Even better is the option to "drag and drop" Opportunities in the desired sorting order. Or is that what you mean by "pin / mark"?

Gr. Gerlof

**Reply by techsmyth** [reply]
Hi Gerlof, 
yes by pinning I mean that you can select a number of opportunities to be fixed in place. 

Related is what kind of priority to give to this? I am hoping to start looking at things like pinning / liking in the coming months, question is if this needs to be accelerated. For now I suggest I capture it properly as a separate epic on the backlog, do initial sizing and then we can look at the whole backlog when that is properly setup.

Fijine avond, Neil

**Reply by GerlofdeHaan** [reply]
Hi Neil,

That's ok with me for now. Users can alway choose "see all" from the dashboard to browse all the Opportunities (and the number off Opportunities in "my" Challenge is restricted to 4, so there is very little chance you lose track..).

Thanks
Gerlof

**Reply by GerlofdeHaan** [reply]
In the latest release sorting bij alphabetical order is implemented. For me this works fine, so I mark this one as answered.

**Comment by GerlofdeHaan** [comment, marked as answer]
In the latest release sorting bij alphabetical order is implemented. For me this works fine, so I mark this one as answered.
